### PR TITLE
Add a completeness error found in Zendoo polynomial inplementation

### PR DESCRIPTION
**Pull request by Mirror-Tang**
## Describe your changes

I add a **completeness error** found in Zendoo polynomial inplementation. Incorrect polynomial representation resulting from arithmetic operations may break assumptions and lead to erroneous computations or may result in denial of service attacks via Rust panics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1356,6 +1357,87 @@ isNeg * (1-isNeg) = 0
...
+## <a name="Zendoo-polynomial">24. Zendoo: Missing Polynomial Normalization after Arithmetic Operations</a>
+
+**Summary**
+
+Related Vulnerabilities: Bad Polynomial Implementation
+
+Identified By: [NCC Group](https://www.nccgroup.com/)
+
+Incorrect polynomial representation
+resulting from arithmetic operations may break as sumptions and lead to erroneous computations or
```
Comment (kcharbo3) [inline README.md:1369]: Typo: Change "as sumptions" to "assumptions"

**Review by kcharbo3 — COMMENTED** [review]
Sorry for the long delay in reviewing this. Looks great though, thanks for adding this! Just had one small nitpick on a typo.

File: README.md (hunk 1 above)
Comment (Mirror-Tang) [reply README.md]: Thank you! I have corrected this spelling error, please check.

Comment (kcharbo3) [reply README.md]: LGTM, thanks!

**Pull request merged**